ios-base/DroidKaigi 2020/SideBar/Views/SidebarViewController.swift

```diff
@@ -0,0 +1,61 @@
+import Material
+import UIKit
+
+enum SwitchViewControllerType: Int {
+    case timeline
+    case about
+    case info
+    case map
+    case sponsor
+    case contributor
+    case setting
+}
```

Comment:
Cool!
I think it would be better to make this `private` or put this in `SidebarViewController` because this name is too generic.